### ISSUE TITLE
[Feature Request] Fuzzing Tab

### ISSUE THREAD

**Issue by joanbono** [ProxymanApp/Proxyman#393]
It would be nice to have a feature or a tab called Fuzz or Fuzzing.

The main idea on this would be to test how an API, for example, responds after multiple requests in a short amount of time.

What I was thinking was something like:
An user is able to select multiple fields and enclose them in special characters (In the example, I'm enclosing it using `$$`, as `$$_whathever_$$`:

![image](https://user-images.githubusercontent.com/7288621/73818320-5cebcd00-47ed-11ea-945a-822c40e022ed.png)

From this point, select or load a txt file with a list:

```sh
# list.txt
shutdown
test1
test2
test3
start
stop
```

And then, after clicking the **Fuzz** button, Proxyman replaces the enclosed fields with the items in the list.

What do you think?

**Comment by NghiaTranUIT** [comment]
I would put it as a Scripting feature, which is in the Road map.

Basically, the end-user could write JS to call the Compose and construct the URL (with Header, URL, Body) and send. It's flexible than hard coding this feature inside Proxyman.

ETA could be in this year.

For now, I suggest that you could write small NodeJS (axios lib) code to automatically do it and make sure set the Proxy to 9090. How do you think?

**Comment by joanbono** [comment]
Sure, thanks! :D

**Comment by NghiaTranUIT** [comment]
Thank. I will push Scripting feature soon 👍

**Comment by shirshak55** [comment]
@NghiaTranUIT Yea we can already use http tools like axios but ability to make plugins for proxyman would solve lots of hassle like this.

**Comment by NghiaTranUIT** [comment]
yes, sure. I receive a lot feature request for the plugin as well as the scripting too. Would support it after the Remote Map, which is on progress now 🙌
